dev-docs/bidders/ampliffy.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: bidder
+title: Ampliffy
+description: Prebid Ampliffy Bidder Adapter
+biddercode: ampliffy
+pbjs: true
+tcfeu_supported: true
+gvl_id: 1258
+media_types: banner, video, native
```

Comment:
If you support multiple media_types please add the following information

- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- safeframes_ok: true/false

This helps others to vet your adapter

Reply:
Thanks! I've added both options.